src/collective/collectionfilter/tiles/__init__.py

```diff
@@ -1,6 +1,18 @@
 # -*- coding: utf-8 -*-
 from plone import api
 from plone.tiles.tile import PersistentTile
+from plone.app.blocks.layoutbehavior import ILayoutBehaviorAdaptable
+from plone.app.blocks.layoutbehavior import ILayoutAware
+import re
+from zope.component import adapter
+from zope.component import getMultiAdapter
+from zope.component import queryAdapter
+from zope.interface import implementer
+from collective.collectionfilter import _
+from collective.collectionfilter.interfaces import ICollectionish
+from collective.collectionfilter.filteritems import CollectionishCollection
+from plone.app.contenttypes.behaviors.collection import ICollection
+from Products.statusmessages.interfaces import IStatusMessage
```

Comment:
we already have `plone.api`... no need for this import ...

Reply:
fixed